src/main/java/seedu/address/model/person/Person.java

```diff
@@ -94,13 +100,14 @@ public boolean equals(Object other) {
                 && otherPerson.getPhone().equals(this.getPhone())
                 && otherPerson.getEmail().equals(this.getEmail())
                 && otherPerson.getAddress().equals(this.getAddress())
-                && otherPerson.getExpectedGraduationYear().equals(this.getExpectedGraduationYear());
+                && otherPerson.getExpectedGraduationYear().equals(this.getExpectedGraduationYear())
+                && otherPerson.getInterviewDate().equals(this.getInterviewDate());
```

Comment:
Will two persons with two different interview date but everything else identical are different reasonble to you? I did not include Resume here since I think it is quite unnatural to distinct person by resume file.

Reply:
Thanks for the comment, but if this is the case we should also change the `hashCode` function to not take this into account

Reply:
But they actually hashed tags as well even though tag is not part of the `equals` function?

Reply:
As per Java 8 documentation: https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#hashCode--

> If two objects are equal according to the equals(Object) method, then calling the hashCode method on each of the two objects must produce the same integer result.

Reply:
ok!